crates/router/src/core/user.rs

```diff
@@ -573,6 +574,9 @@ pub async fn invite_user(
         let is_email_sent;
         #[cfg(feature = "email")]
         {
+            // Doing this to avoid clippy lints
+            // will add actual usage for this later
+            let _ = req_state.clone();
```

Comment:
Maybe add a `// TODO` instead? It's easy for this to get lost otherwise.

Reply:
added usage in [this](https://github.com/juspay/hyperswitch/pull/4166) PR